src/index.html

```diff
@@ -10,8 +10,213 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="label">
+          Surname:
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="name" class="label">
+          Name:
+          <input
+            id="name"
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="age" class="label">
+          How old are You?
+          <input
+            id="age"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="dateOfBirth" class="label">
+          Full date of birth:
+          <input
+            id="dateOfBirth"
+            type="date"
+            name="dateOfBirth"
+            placeholder="mm/dd/yyyy"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+
+
+        <label for="acceptTerms" class="label">
```

Comment:
```suggestion
        </label>
        
        <label for="acceptTerms" class="label">
```
one empty line is enough